tests/migrations/test_loader.py

```diff
@@ -367,3 +369,19 @@ def num_nodes():
         self.assertEqual(num_nodes(), 0)
 
         recorder.flush()
+
+    @override_settings(MIGRATION_MODULES={'migrations': 'migrations.test_migrations_inconsistent_history'})
+    def test_check_consistent_history(self):
+        """Loading migrations with inconsistent history should error."""
+        loader = MigrationLoader(connection=None)
+        loader.check_consistent_history(connection)
+        recorder = MigrationRecorder(connection)
+        recorder.record_applied('migrations', '2_auto')
+
+        msg = (
+            "Migration ('migrations', '2_auto') is applied before its "
```

Comment:
Nitpicking:
The current migration output uses both `('app_label', 'migration_name')` and `app_label.migration_name` scattered all over the place. I'll open a ticket to use the dot-notation across the board which I find easier to read / write. In the mean time I'd use that format here already.